README.md

```diff
@@ -1,148 +1,213 @@
+easy and elegant.
+
+Crate provides several installation packages, including a supported
+Docker image. It fits perfectly in an orchestrated microservices
+environment. It acts like an ether, an omnipresent, persistent layer
+for data. This way, application containers access their data
+regardless on which host the data nodes run.
+
+[Crate: Put your data to work. Simply.](https://crate.io/)
+
+## Building Images
+
+Crate derives from
+[alpine:latest](https://hub.docker.com/_/alpine/). We build this
+nightly updated image from the `Dockerfile` on GitHub:
 
-We solve your data scaling problems and make administration a breeze.
-
-**Crate.IO - Put your data to work. Simply.**
-
-[http://crate.io][3]
-
-
-# Crate Dockerfile
-
-This repository contains **Dockerfile** of [Crate][3] for [Docker][1]'s [automated build][2]
-published to the public [Docker Hub Registry][4].
-
-
-## Base Docker Image
-
-- [alpine:latest][5]
-
-## Installation
-
-1. Install [Docker][1]
-
-2. Download latest automated build from public [Docker Hub Registry][2]
+    docker build -t="crate/crate" github.com/crate/docker-crate
 
-    docker pull crate/crate:latest
+To build your own image, replace the image name after the `-t` option
+with your own name.
 
-Alternatively you can build an image from `Dockerfile`:
-
-    docker build -t="crate/crate" github.com/crate/docker-crate
+Alternativly, an [Official Crate
+Image](https://hub.docker.com/_/crate/) with the name `crate` is
+available in the Docker Hub. That image is based on a stable major
+version of Crate, and is tested and scanned by Docker, Inc.
 
 ## How to use this image
 
-    docker run -d -p 4200:4200 -p 4300:4300 crate/crate:latest
+To form a cluster, just start the Crate container a few times in the
+background. This starts a couple of containers on your machine which
+discover each other via multicast. In a production environment you'd
+run Crate on different machines:
 
-### Attach persistent data directory
+```console
+# docker run -d crate/crate
+```
 
-    docker run -d -p 4200:4200 -p 4300:4300 -v <data-dir>:/data crate/crate
+To access the admin UI, point your browser to port tcp/4200 of a data
+node of your choice while you start it or look up its IP later on:
 
-### Use custom Crate configuration
+```console
+# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate/crate)):4200/admin"
+```
 
-    docker run -d -p 4200:4200 -p 4300:4300 crate/crate crate -Des.config=/path/to/crate.yml
+## Attach persistent data directory
 
-Any configuration settings may be specified upon startup using the `-D` option prefix.
-For example, configuring the cluster name by using system properties will work this way::
+Crate stores all important data in `/data`. To attach it to a backup
+service, map the directory to the host:
 
-    docker run -d -p 4200:4200 -p 4300:4300 crate/crate crate -Des.cluster.name=cluster
+```console
+# docker run -d -v <data-dir>:/data crate/crate
```

Comment:
Using `-v` is just an example of how to make your node more persistant under the assumption your containers are ephemeral, but your hosts aren't. Leaving out `-p` works here under the same assumptions as above.